lib/framereader.py

```diff
@@ -165,6 +165,28 @@ def read_frame_P210(fd, width, height):
 
   return y, u, v
 
+def read_frame_Y210(fd, width, height):
+  #https://docs.microsoft.com/en-us/windows/win32/medfound/10-bit-and-16-bit-yuv-video-formats#422-formats
+  #Y210 (each pair of pixels is stored as an array of four WORD)
+  #   Y0           U            Y1             V
+  #  word0        word1        word2         word3
+  #bits 15-0     bits15-0     bits15-0      bits15-0
+  #
+  #The 16bit representations described here use little-endian WORD values for each channel
+  #bits:  15 14 13 ...  ...     5 4 3 2 1 0
+  #data:  valid data  ...       0 0 0 0 0 0
+  width2  = (width + 1) // 2
+  size    = width * height
+  size2   = width2 * height * 4
+
+  y210 = numpy.fromfile(fd, dtype=numpy.uint16, count=size2)
```

Comment:
count = `size * 2`?

Reply:
Yes, this is using size2, not type issue,
Based on https://docs.microsoft.com/en-us/windows/win32/medfound/10-bit-and-16-bit-yuv-video-formats#422-formats The Y210 define each pair of pixels is stored as an array of four WORD,  (that is 8 16bit for two pixels)
On here, the numpy read unit is uint16, so size is size2=((width+1)/2) * height * 4

but there is no using for "size    = width * height" define, let me to remove this one, and rename the size2 to size;